sycl/source/detail/memory_manager.cpp

```diff
@@ -219,7 +219,7 @@ MemoryManager::allocateBufferObject(ContextImplPtr TargetContext, void *UserPtr,
     /// out a better way to enforce buffer DDR bank assignments, lazy rather
     /// than eager buffer creation?
     cl_mem_ext_ptr_t mext = {0};
-    mext.banks = 0 | XCL_MEM_TOPOLOGY;
+    //mext.banks = 0 | XCL_MEM_TOPOLOGY;
```

Comment:
Why?

Reply:
Actually the whole Xilinx specfic ifdef can be removed as XRT is able to figure out which bank ram is associated with which kernel.
The commented line forced to allocate the buffer in DDR bank 0.